<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>551</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ServerIP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtServerIP">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPort">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>ConnCount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbConnCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="txtData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>541</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendData">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
